bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/ISharedImages.java

```diff
@@ -678,16 +741,98 @@ public interface ISharedImages {
 	 *
 	 * @see Cursor#Cursor(Device, ImageData, ImageData, int, int)
 	 * @since 3.5
+	 * @deprecated Dropped in favor of {@link #IMG_OBJS_DND_TOFASTVIEW}. The
+	 *             replacement icon does <b>not</b> require a mask.
+	 * @noreference Marked for deletion since newer versions are available; see
+	 *              https://github.com/eclipse-platform/eclipse.platform.ui/pull/2084
 	 */
+	@Deprecated(forRemoval = true, since = "2024-09")
 	String IMG_OBJS_DND_TOFASTVIEW_SOURCE = "IMG_OBJS_DND_TOFASTVIEW_SOURCE"; //$NON-NLS-1$
 	/**
 	 * Cursor 'mask' for the 'fast-view' arrow cursor. For cursor construction see:
 	 *
 	 * @see Cursor#Cursor(Device, ImageData, ImageData, int, int)
 	 * @since 3.5
+	 * @deprecated No longer needed with {@link #IMG_OBJS_DND_TOFASTVIEW}.
+	 * @noreference Marked for deletion since newer versions are available; see
+	 *              https://github.com/eclipse-platform/eclipse.platform.ui/pull/2084
 	 */
+	@Deprecated(forRemoval = true, since = "2024-09")
 	String IMG_OBJS_DND_TOFASTVIEW_MASK = "IMG_OBJS_DND_TOFASTVIEW_MASK"; //$NON-NLS-1$
 
+	// The following set of constants represent the image pairs that are used
+	// to construct cursors for drag and drop operations within the workbench
+	// Each cursor is represented by two images; the 'source' and the 'mask'
+	// These need to be combined using the following code snippet:
+	// source = getImageDescriptor(sourceId);
+	// mask = getImageDescriptor(maskId);
+	// cursor = new Cursor(display, source.getImageData(), mask.getImageData(), 16,
+	// 16);
```

Comment:
This seems to be an unintended copy. The following constants represent icons that work standalone without a source/mask combination.

Reply:
That is true. I initially wanted to emphasize that the hotspot of those cursors is the center of the image and thus needs to be calculated based on the current device zoom. I simply forgot to push those changes but have done so, now.